docs/content/v0.20/packaging.md

```diff
@@ -231,10 +243,15 @@ built from.
 
 ### KubeFlex PostCreateHooks
 
-There are two `PostCreateHook` objects defined in [config/postcreate-hooks](../../../config/postcreate-hooks). What uses these copies of those hook definitions?
+There are two `PostCreateHook` objects defined in [config/postcreate-hooks](../../../config/postcreate-hooks).
+
+#### ocm PostCreateHook
+
+The PostCreateHook defined in `ocm.yaml` gets used on an ITS and adds the hub side of OCM there, using the image `quay.io/kubestellar/clusteradm:0.7.2`. See [above](#clusteradm-container-image) about the source of that. Currently this PostCreateHook is used in the E2E tests but this is a problem because of its fixed reference to container image previously built from sources in this same Git repository.
+
+#### kubestellar PostCreateHook
 
-- `ocm.yaml` adds `clusteradm` by running a container using the image `quay.io/kubestellar/clusteradm:0.7.2`; see [above](#clusteradm-container-image) about the source of that.
-- `kubestellar.yaml` runs container image `quay.io/kubestellar/helm:v3.14.0` (which is built from [the Helm source](https://github.com/helm/helm/tree/v3.14.0) by a process that we need to document) to instantiate the chart from `oci://ghcr.io/kubestellar/kubestellar/kubestellar-operator-chart` with chart version `0.20.0-alpha.1`, which is built by (what?) from (what? probably answered above).
+The PostCreateHook defined in `kubestellar.yaml` is intended to be used in the hosting cluster, once per WDS, and runs container image `quay.io/kubestellar/helm:v3.14.0` (which is built from [the Helm source](https://github.com/helm/helm/tree/v3.14.0) by a process that we need to document) to instantiate the chart from `oci://ghcr.io/kubestellar/kubestellar/kubestellar-operator-chart` with chart version `0.20.0-alpha.1`. Currently nothing uses this PostCreateHook (the [examples doc](examples.md) reports on a past use of a previous build of this PostCreateHook).
 
```

Comment:
We have not changed this PCH since 0.20.0-alpha.1 - the current example doc in main still references it since it is based on the latest snapshot available ( 0.20.0-alpha.1  ).  Does " Currently nothing uses this PostCreateHook" mean to say that no automated tests are using it?

Reply:
No, it means to say that nothing references the copy of this PCH in `main` ; examples.md references the copy in the commit tagged `v0.20.0-alpha.1`.

Reply:
I have revised to hopefully make it clearer.